CostumersAPI/Services/CostumersService.cs

```diff
@@ -0,0 +1,67 @@
+﻿using CostumersAPI.Services.Interfaces;
+using CostumersAPI.Costumer;
+using FluentValidation;
+using System.Collections.Generic;
+using System;
+using System.Linq;
+
+namespace CostumersAPI.Services
+{
+    public class CostumersService : ICostumerService
```

Comment:
```suggestion
    public class CustomerService : ICustomerService
```
O correto é Customer, e no singular

Reply:
Acabei criando o projeto com a escrita errada, mas já fiz a correção durante o 2º desafio.